git-hub

```diff
@@ -838,15 +838,13 @@ class CloneCmd (object):
                     repo['full_name'])
         else:
             upstream = proj
-            for repo in req.get('/repos/' + proj + '/forks'):
-                if repo['owner']['login'] == config.username:
-                    break
-            else: # Fork not found
-                infof('Forking {} to {}', upstream,
-                    config.username)
-                repo = req.post('/repos/' + upstream + '/forks')
-                infof('New fork at {}', repo['html_url'])
-            repo = req.get('/repos/' + repo['full_name'])
+            # Try to fork, if a fork already exists, we'll get the
+            # information for the pre-existing fork
+            # XXX: This is not properly documented in the GitHub
+            #      API docs, but it seems to work as of Sep 2016
+            infof('Checking for fork...')
```

Comment:
Yeah, the messages got a bit messy because of this, specially because we can't really tell if we just forked or not. But I guess "Checking for existing fork/forking..." could make sense.